src/main/waveform/waveform.mixins.js

```diff
@@ -35,7 +35,9 @@ define(['konva'], function (Konva) {
 
       // If no color is provided for the marker, fall back to the segment's color
       if (color === undefined) {
-        color = parent.color;
+        useColor = parent.color;
```

Comment:
(You don't have to set the variable to anything. You just need to say var X to prevent setting a global variable.)